lib/ansible/modules/source_control/git.py

```diff
@@ -1019,6 +1019,14 @@ def main():
         dest = os.path.abspath(dest)
         if bare:
             gitconfig = os.path.join(dest, 'config')
+        elif os.path.isfile(os.path.join(dest, '.git')):
+            with open(os.path.join(dest, '.git'), 'r') as gitfile:
+                data = gitfile.read()
+                match = re.match('gitdir: (.+)$', data)
```

Comment:
Please dedent all the following code, so that it would be outside of `with`-block.